blazor/rating/selection.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: Selection in Blazor Rating Component | Syncfusion
+description: Checkout and learn here all about Selection with Syncfusion Blazor Rating component in Blazor Server App and Blazor WebAssembly App.
+platform: Blazor
+control: Rating
+documentation: ug
+---
+
+# Selection in Blazor Rating Component
+
+All the items before the selected item should be in selected state in a rating component.
+
+Below example demonstrates Selection of Rating.
+
+```cshtml
+
+@using Syncfusion.Blazor.Inputs
+
+<SfRating Value="3"></SfRating>
+
+```
+
+![Blazor Rating Component with Selction](./images/blazor-rating-selection.png)
+
+## Selected Value
+You can get the current updated rating item value selected by the user by setting [Value](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Inputs.SfRating.html#Syncfusion_Blazor_Inputs_SfRating_Value) property.
+
+Below example demonstrates the selected value of Rating.
+
+```cshtml
+
+@using Syncfusion.Blazor.Inputs
+
+<SfRating @bind-Value=value></SfRating>
+<br/>
+Value Is : @value
+@code{
+    double value= 3;    
+}
+
+```
+
+![Blazor Rating Component with Selected Value](./images/blazor-rating-selected-value.png)
+
+## Min Value
+
+You can customize the minimum possible rating value by setting [Min](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Inputs.SfRating.html#Syncfusion_Blazor_Inputs_SfRating_Min) property. If you want to allow users to provide ratings from 2, you can set the Min property to 2.
+
+Below example demonstrates the Min Value of Rating.
+
+```cshtml
+
+@using Syncfusion.Blazor.Inputs
+
+<SfRating Min="2"></SfRating>
+
+```
+
+![Blazor Rating Component with Label Template](./images/blazor-rating-min.png)
+
+## Single Selection
+
+You can select only a single rating item by setting [EnableSingleSelection](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Inputs.SfRating.html#Syncfusion_Blazor_Inputs_SfRating_EnableSingleSelection) property. Only the selected item will be in the selected state, and all other items will be un-selected.
+
+Below example demonstrates the Single Selection of Rating.
+
+```cshtml
+
+@using Syncfusion.Blazor.Inputs
+
+<SfRating EnableSingleSelection=true></SfRating>
+
+```
+
+![Blazor Rating Component with Single Selection](./images/blazor-rating-single-selection.png)
+
+## Show Or Hide Reset Button
```

Comment:
start with capital letter for first word alone

Reply:
I have updated it as per comments.